src/main/java/com/icthh/xm/uaa/web/rest/UserResource.java

```diff
@@ -232,6 +236,14 @@ public ResponseEntity createUser(@Valid @RequestBody UserDTO user) throws URISyn
         return new ResponseEntity<>(page.getContent(), headers, HttpStatus.OK);
     }
 
+    @GetMapping("/users/filter")
+    @Timed
+    public ResponseEntity<List<UserDTO>> getAllByFilers(@ApiParam Pageable pageable, UserFilterQuery userFilterQuery){
```

Comment:
@ibilous I think we need to add some permission here.

Reply:
added @PostAuthorize